tests/lib/virt.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import libvirt
+import time
+
+import xml.etree.ElementTree as ET
+
+from lib.log import log
+
+# filled in by connectDomain function
+snapshots = None
```

Comment:
Wrapped into class, one instance per domain (not enforced in the library in any way, so it could potentially make mess, but imho point is not to create bulletproof library, but to create a tool in early stages of production)